trigger-deployment.js

```diff
@@ -0,0 +1,43 @@
+const {
+  // (this comment is needed to prevent Prettier from merging the lines below)
```

Comment:
Could we create `scripts` directory and move this script inside?

Reply:
Done in 408d18d